java-checks/src/test/files/checks/IndentationCheck_default.java

```diff
@@ -1,6 +1,18 @@
 import java.util.stream.IntStream;
 
 class TestSwitch {
+
+  public record QonId(String value) {
+      public QonId { // Noncompliant
+      Objects.requireNonNull(value, "QonId value cannot be null");
+    }
+  }
+
+  public record QonId_(String value) {
+    public QonId_ { // compliant
+      Objects.requireNonNull(value, "QonId value cannot be null");
+    }
+  }
```

Comment:
It looks like we added this test case to a non-compiling file. Let's move that file to `java-checks-test-sources/src/main/java/checks` or at least `java-checks-test-sources/src/main/files/non-compiling/checks`

Reply:
:+1: